packages/core/src/components/GlobalActions/GlobalActions.styles.tsx

```diff
@@ -20,9 +20,8 @@ export const { staticClasses, useClasses } = createClasses("HvGlobalActions", {
     "&:before": {
```

Comment:
We no longer see the `::before` pseudo element behind the main component to blur the background like this: 

https://github.com/lumada-design/hv-uikit-react/assets/43220251/57b92785-3e29-4a3c-b246-a6a2f80c8e00


I believe something like this would fix it (I removed the `marginTop` from `globalWrapperComplement` and changed `global`):

```
global: {
    zIndex: `calc(${theme.zIndices.banner} - 2)`,
    top: 0,
    left: 0,

    paddingTop: theme.space.xs,
    paddingBottom: theme.space.xs,

    "&:before": {
      content: "''",
      display: "flex",
      width: "100%",
      background: theme.colors.atmo2,
      opacity: "75%",
      
      height: "100%",
      top: 0,
      position: "absolute",
    },
    backdropFilter: "blur(1px)",
  },
  globalWrapperComplement: {
    position: "relative",
    top: 0,
    left: 0,
    background: theme.colors.atmo1,
    width: "100%",
  },
```

Reply:
I've changed it to `padding` in `global` 👍